code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -384,3 +384,20 @@
 		return
 
 	return max((maxHealth - health), 0)
+
+//version for the infestation event
+/mob/living/carbon/alien/diona/ghost_playable
+	var/playable = TRUE
+	thinking_enabled = TRUE
```

Comment:
why is thinking_enabled set to TRUE here when you have the mob stop thinking anyway when it's spawned in the ghost spawner?